### ISSUE TITLE
unersolved symbol in :inline tag

### ISSUE THREAD

**Issue by bsless** [borkdude/clj-kondo#635]
**version**

clj-kondo v2019.10.26

**platform**

native

**editor**

Emacs 27.0.5 `87347a5bbc2f044c51feea8d513fb1dcefa6f50e`
Spacemacs develop `bc22a259e87ed6ff3a1fcdb83dd27ffacad77a15`
Clojure settings:
```elisp
(clojure :variables
              clojure-enable-linters '(squiggly joker clj-kondo)
              clojure-enable-clj-refactor t
              clojure-enable-fancify-symbols t)
```

**problem**

Kondo reports unresolved symbol for an `:inline` meta tag with inline definition.

**repro**

repro.clj:
```clojure
(ns repro)

(def that-name "Kondo!")

(defn that-name?
  ""
  {:tag Boolean
   :static true
   :inline (fn [s] (list 'clojure.lang.Util/identical s that-name))}
  [s]
  (= that-name s))
```
kondo output:
```
repro.clj:9:17: error: unresolved symbol s
```

**expected behavior**

I expect `s` to be recognized as a binding

**Comment by borkdude** [comment]
Can you upgrade to the latest version? I believe this was already fixed.

**Comment by bsless** [comment]
that fixed it, thank you!
